drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/DetectImpossibleMatchCheck.java

```diff
@@ -67,21 +68,20 @@ public Issue getIssue() {
             fieldName = field.getName();
             fieldFactType = field.getFactType();
         }
-        String conflictedItem = toHumanReadableString( conflict.getOrigin()
-                                                  .getConflictedItem() );
-        String conflictingItem = toHumanReadableString( conflict.getOrigin()
-                                                   .getConflictingItem() );
-
-        Issue issue = new ImpossibleMatchIssue( Severity.ERROR,
-                                                ExplanationType.IMPOSSIBLE_MATCH,
-                                                Integer.toString( ruleInspector.getRowIndex() + 1 ),
-                                                fieldFactType,
-                                                fieldName,
-                                                conflictedItem,
-                                                conflictingItem,
-                                                ruleInspector.getRowIndex() + 1 );
 
+        return new ImpossibleMatchIssue( Severity.ERROR,
+                                         checkType,
+                                         Integer.toString( ruleInspector.getRowIndex() + 1 ),
+                                         fieldFactType,
+                                         fieldName,
+                                         toHumanReadableString( conflict.getOrigin()
+                                                                        .getConflictedItem() ),
+                                         toHumanReadableString( conflict.getOrigin()
+                                                                        .getConflictingItem() ),
+                                         ruleInspector.getRowIndex() + 1 );
+    }
 
-        return issue;
+    public static CheckType getType() {
```

Comment:
@Rikkola this method is kind of suspicious. I can not find another use of this method in the PR? Is it intended?